### PR TITLE
Feature/ Implement input fields component

### PR DESCRIPTION
**Pull request by JennyDoJD**
Result (updated):
Case: No error
image

image

Case: Error
image

Ticket: [Implement InputFields common component](https://trello.com/c/0xfMMNk8/27-implement-inputfields-common-component)

### CODE REVIEW

File: practice/react-practice-one/src/components/InputFields/InputField.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/* Import dependencies */
+import PropTypes from 'prop-types';
+
+/* Import CSS */
+import './inputField.css';
+
+const InputField = ({ type }) => {
+  return <input type={type} className="input-container" />;
```
Comment (ghost) [inline practice/react-practice-one/src/components/InputFields/InputField.jsx]: How about `TextField`?

Comment (ghost) [reply practice/react-practice-one/src/components/InputFields/InputField.jsx]: @JennyDoJD This component should have more props. 
Will you use this component in ProductF from? If yes, pls consider the props need in this component

Comment (ghost) [reply practice/react-practice-one/src/components/InputFields/InputField.jsx]: how about an error case? input should be displayed a message
Refer https://mui.com/material-ui/react-text-field/

**Comment by JennyDoJD** [comment]
Result (updated):
Case: No error
image

image

Case: Error
image

**Review by ghost — APPROVED** [review]

**Pull request merged**